src/main/scala/exercises/07_Palindrome.scala

```diff
@@ -8,6 +8,24 @@ object Palindrome {
    *  Hint: the Scala collections API is your friend
    *  http://docs.scala-lang.org/overviews/collections/overview.html 
    */
-  def isPalindrome(s: String): Boolean = throw new NotImplementedError
+  def isPalindrome(s: String): Boolean = {
+   	//filter out the non-letter characters
+  	val letters = s.filter(a => a.isLetter)
+  	// make all the letters lowercase
+  	val lowercase = letters.toLowerCase()
+
+  	isPalindromeHelper(lowercase)
+  }
+
+  /**
+  * Given a string of lowercase letters, ensures that it is a palindrome
+  */
+  def isPalindromeHelper(s: String): Boolean = {
```

Comment:
This is a really interesting way of doing it. I just reversed the strings and output whether or not they were equal, but this definitely works as well.